datashuttle/tests/test_configs.py

```diff
@@ -48,11 +48,112 @@ def test_warning_on_startup(self):
             == "Configuration file has not been initialized. Use make_config_file() to setup before continuing."
         )
 
+    def test_fail_to_pass_remote_path(self, project):
+        """
+        Test that the make_config_file will assert if neither
+        remote_path_ssh or remote_path_local are passed.
+        """
+        try:
+            project.make_config_file("test_local_path", False)
+            assert False, "expected error was not caught"
+
+        except AssertionError as e:
```

Comment:
Would [`pytest.raises()`](https://docs.pytest.org/en/stable/reference/reference.html?highlight=raises#pytest.raises) be simpler for testing these errors?